### ISSUE TITLE
[Spike: 2h] Investigate Grafana alerting for servers failing to start-up

### ISSUE THREAD

**Issue by sgibson91** [2i2c-org/infrastructure#4604]
### Context

The most common symptom of an outage is servers failing to spin up. We have simple uptime checks that tell us whether the hub is up and reachable, but this doesn't mean it's _useful_. Hence we often find out about outages via support tickets reporting that servers can't start. Ideally, we should find out about this problem before someone has to email us, as that is usually a critical time ("My class starts in 10 mins...") and puts extra pressure on the engineering team to resolve it quickly. If we catch the outage earlier, we buy ourselves more time to investigate and fix the issue, and hopefully can resolve it before a user realises there was ever a problem :)

- We already track server start failures in grafana: https://grafana.pilot.2i2c.cloud/goto/qaf3fq9Ig?orgId=1
- Grafana also supports alerting: https://grafana.com/docs/grafana/latest/alerting/

We should perform a spike to understand if alerting from grafana on failed server starts is possible, and what would be required to implement this.

### Definition of Done

- We understand whether or not alerting from grafana on failed server starts is possible
- If possible, we have enough understanding to scope an epic and associated tasks to implement this

**Comment by yuvipanda** [comment]
Looking at the internal roadmap, I see 3 other items ahead of this:

- https://github.com/2i2c-org/meta/issues/1239
- https://github.com/2i2c-org/infrastructure/issues/4390
- https://github.com/orgs/2i2c-org/projects/54/views/4?pane=issue&itemId=70093903

The internal roadmap board has a WIP limit of 2, so I'd like us to wait for some of these to be completed before this gets picked up.

**Comment by sgibson91** [comment]
Yes, I did bring that up in the planning meeting because we were just going to add everything to Committed as we knew we'd get more capacity from Erik next week. But I didn't realise that would mean it needed to go all the way back to Backlog.
